bearer/bearer.go

```diff
@@ -63,6 +63,11 @@ func (b *Generator) NewBearer(email string, currentEpoch uint64, msPerEpoch int6
 	eacl.AddFormedTarget(rec2, eacl.RoleOthers)
 	t.AddRecord(rec2)
 
+	rec3 := eacl.CreateRecord(eacl.ActionDeny, eacl.OperationPut)
```

Comment:
It has to come _before_ `rec`. If `rec` matches this is an unreachable rule.

Reply:
Moved